src/tensor_ops/fast_gelu/mod.rs

```diff
@@ -0,0 +1,79 @@
+mod cpu_kernel;
+
+#[cfg(feature = "cuda")]
+mod cuda_kernel;
+
+use super::ops::{try_unary_op, UnaryKernel};
+use crate::{shapes::*, tensor::*};
+
+#[allow(unused)]
+#[deprecated(since = "0.12.0", note = "use `FastGeLUKernelOp` instead")]
+pub type GeLUKernelOp = FastGeLUKernelOp;
+
+#[repr(C)]
+#[derive(Debug, Default, Copy, Clone)]
+pub struct FastGeLUKernelOp;
+
+/// [Fast Gaussian Linear Unit (GeLU)](https://paperswithcode.com/method/gelu). `0.5 * x * (1 + tanh(sqrt(2 / pi) * (x + 0.044715 * x^3)))`
+///
+/// Examples:
+/// ```rust
+/// # use dfdx::prelude::*;
+/// # let dev: Cpu = Default::default();
+/// let t = dev.tensor([-1.0, 0.0, 1.0, 2.0]);
+/// let r = t.gelu();
+/// ```
+pub fn fast_gelu<S: Shape, E: Dtype, D: UnaryKernel<FastGeLUKernelOp, E>, T: Tape<E, D>>(
+    t: Tensor<S, E, D, T>,
+) -> Tensor<S, E, D, T> {
+    t.fast_gelu()
+}
+
+#[deprecated(since = "0.12.0", note = "Use `fast_gelu` instead")]
+pub fn gelu<S: Shape, E: Dtype, D: UnaryKernel<FastGeLUKernelOp, E>, T: Tape<E, D>>(
+    t: Tensor<S, E, D, T>,
+) -> Tensor<S, E, D, T> {
+    fast_gelu(t)
+}
+
+impl<S: Shape, E: Dtype, D: UnaryKernel<FastGeLUKernelOp, E>, T: Tape<E, D>> Tensor<S, E, D, T> {
+    /// See [gelu]
+    pub fn fast_gelu(self) -> Self {
+        self.try_fast_gelu().unwrap()
+    }
+    /// See [gelu]
+    pub fn try_fast_gelu(self) -> Result<Self, D::Err> {
+        try_unary_op(FastGeLUKernelOp, self)
+    }
+
+    #[deprecated(since = "0.12.0", note = "Use `fast_gelu` instead")]
+    pub fn gelu(self) -> Self {
+        self.fast_gelu()
+    }
+
+    #[deprecated(since = "0.12.0", note = "Use `try_fast_gelu` instead")]
+    pub fn try_gelu(self) -> Result<Self, D::Err> {
+        self.try_fast_gelu()
+    }
```

Comment:
Top two methods should link to `fast_gelu`, and deprecated items should have a link to their non-deprecated counterparts.